ODIN_II/SRC/netlist_check.cpp

```diff
@@ -717,3 +721,90 @@ nnode_t* find_node_at_top_of_combo_loop(nnode_t* start_node) {
         }
     }
 }
+
+/**
+ * @brief This function travseres the netlist both forwards and backwards and keeps the record of nodes with same type in each traverse.
+ * In case that there is a mismatch between the number of nodes counted in both traverses, It will throw a warning to
+ * indicate a discontinuity in the netlist.
+ */
+void check_discontinuity(netlist_t* netlist) {
+    //Declare variables that hold number of elements
+    long long num_of_forward_elements{0}, num_of_backward_elements{0};
+    //Declare type of the node
+    operation_list op{MULTIPLY};
```

Comment:
Functions and methods are better be written in a general format, not explicitly determined for a specific node/operation type. Please consider writing it in a way that it can report for all types.

Reply:
The function now reports each type of removed nodes.